atramhasis/views/views.py

```diff
@@ -79,6 +79,13 @@ def home_view(self):
 
         return {'conceptschemes': conceptschemes}
 
+    @view_config(route_name='conceptscheme_tree', renderer='scheme_tree.jinja2')
+    def scheme_tree_view(self):  # pragma: no cover
+        """
+        duplicates view of the beeldbank application
```

Comment:
kweet niet of de comment hier intended is?

Reply:
Dat lijkt mij niet juist nee :p